### PR TITLE
Simplify wallettemplate/build.gradle

### PR DESCRIPTION
**Pull request by msgilligan**
* Put all Gradle plugins in plugins block
* Use “11” for sourceCompatibility (not “1.11”)

### CODE REVIEW

File: wallettemplate/build.gradle [hunk 1]
```diff
@@ -1,12 +1,10 @@
 plugins {
-    id 'org.openjfx.javafxplugin' version '0.0.7' apply false
```
Comment (schildbach) [inline wallettemplate/build.gradle]: This change looks good to me. To be consistent across all subprojects, could you change it for the other subprojects too?

File: wallettemplate/build.gradle [hunk 2]
```diff
@@ -20,7 +18,7 @@ javafx {
     modules = [ 'javafx.controls', 'javafx.fxml' ]
 }
 
-sourceCompatibility = 1.11
```
Comment (schildbach) [inline wallettemplate/build.gradle]: I'm not so sure about this change. I know OpenJDK 11 supports this syntax, but I believe OpenJDK 8 does not. And I'd like to keep the type of syntax used consistent with the other subprojects.

**Comment by schildbach** [comment]
Ah, drats! This syntax seems to require a newer Gradle version that 3.4.1.

**Comment by schildbach** [comment]
I propose to merge this after we've upgraded to Gradle 4.4.1. Which we should be able to do pretty soon – Gradle 4.4.1 is about to enter Ubuntu 18.04.

**Comment by msgilligan** [comment]
Sorry, I've been testing with Gradle 4.3.10 (the final version of 4.x) but that isn't old enough. It would be nice if CI could somehow test with 3.4.1. (This is where the wrapper comes in handy, but we might be able to do it with a custom .yml file)

**Comment by schildbach** [comment]
On F-Droid, we modify the local copy of build.gradle like this:

> `echo 'task wrapper(type: Wrapper) { gradleVersion = "3.4.1" }' >> ../build.gradle`

And then I think gradlew is executed by the build daemon. Maybe we can adapt that pattern to Travis?

**Comment by msgilligan** [comment]
> Maybe we can adapt that pattern to Travis?

We can give it a try.

I created PR #1781 so that our minimum Gradle requirements are better documented and enforced.

BTW, I use the [sdkman](https://sdkman.io) for my testing so I can do things like:
```
$ sdk use gradle 3.3
$ gradle clean build
$ sdk use java 8.0.201-oracle
$ gradle clean build
$ sdk use gradle 3.4
$ gradle clean build
$ sdk use gradle 4.10.3
$ gradle clean build
$ sdk use java 12.0.0-open
$ gradle clean build
```

**Comment by schildbach** [comment]
I plan to merge this as soon as https://github.com/bitcoinj/bitcoinj/pull/1789 is agreed and merged.

**Comment by schildbach** [comment]
Merged.

**Pull request closed without merging**